src/languageSupport/languages/flux/lsp/utils.ts

```diff
@@ -147,4 +153,5 @@ export enum ExecuteCommand {
   InjectField = 'injectFieldFilter',
   InjectTag = 'injectTagFilter',
   InjectTagValue = 'injectTagValueFilter',
+  CompositionInit = 'composition/initialize',
```

Comment:
We'll need to support both the old and new style of schema injection. (Feature flagged for new schema composition.)

This means that the execute_command() method in the LSP server (receiving the `workspace/executeCommand` payloads) will need to be extended to handle other methods.

Here I placed an example `'composition/initialize'` method. But we can make it be whatever you suggest @rockstar .

Reply:
He already has that whole LSP spec, which states the methods to use.

Followup PR will use those methods. That PR will simply be:
* in the `LspConnectionManager` methods, write a few lines to:
(1) construct the json blob, with the method (per spec)
(2) use existing `LspConnectionManager.inject()` to send to LSP.
(3) and that's it folks.